lib/slack-rtmapi.rb

```diff
@@ -1,14 +1,20 @@
-require_relative "slack-rtmapi/client"
-require_relative "slack-rtmapi/version"
+# encoding: utf-8
+
+require_relative 'slack-rtmapi/client'
+require_relative 'slack-rtmapi/version'
 
 require 'json'
 require 'net/http'
 
-module SlackRTM
-
+#
+# SlackRTM
+#
```

Comment:
I am not fond of empty comment. Either we document, either we do not. I'll push some doc this evening